tests/kernel/boot_ltp.pm

```diff
@@ -37,6 +37,11 @@ sub run {
         assert_script_run("lsblk -la; export LTP_BIG_DEV=$block_dev");
     }
 
+    # check kGraft patch if KGRAFT=1
+    if (check_var('KGRAFT', '1')) {
```

Comment:
Yeah, you should really use `get_var()` or is there a specific reason you check for equality?